docs/design/theming.md

```diff
@@ -52,3 +52,24 @@ div {
   color: var(--theme-color, $black);
 }
 ```
+
+Note that fallback values aren't used to fix the browser compatibility. If the browser doesn't support CSS custom Properties, the fallback value won't help. To prevent this issue on older browsers, we need to write our styles like this:
+
+```scss
+div {
+  color: $black;
+  color: var(--theme-color, $black);
+}
+```
+
+This can be too much work and browser fallback can be forgotten. For a better developer experience, you should use the 2 minxins defined in `app/assets/stylesheets/_mixins.scss` called `themeable` and `themeable-important`. They take 3 arguments. The first argument is the CSS property like `color` or `background`, the second argument is the CSS custom property name (without `--`) like `theme-color` and the third argument is the CSS value like `$black` or `white`.
```

Comment:
Great contribution! just a small typo here: `minxins` instead of `mixins`

Reply:
Nice Catch :+1: